quipucords/scanner/openshift/api.py

```diff
@@ -192,9 +194,15 @@ def retrieve_cluster(self, **kwargs) -> OCPCluster:
     def retrieve_acm_metrics(self, **kwargs) -> list:
         """Retrieve metrics on acm managed clusters."""
         acm_metrics = []
-        for cluster in self._list_managed_clusters(**kwargs).items:
-            managed_cluster_metrics = self._init_managed_cluster(cluster)
-            acm_metrics.append(managed_cluster_metrics)
+        try:
+            for cluster in self._list_managed_clusters(**kwargs).items:
+                managed_cluster_metrics = self._init_managed_cluster(cluster)
+                acm_metrics.append(managed_cluster_metrics)
+        except ResourceNotFoundError:
+            logger.info(
+                _("OpenShift host at %(host)s does not have the ManagedCluster API."),
```

Comment:
We don't need to translate logs (and parts of the code that are doing it, should stop doing it). 

IMO the only things that should be translated are API responses that the UI displays to users.

Reply:
Whoops! I will amend and force push to remove the unnecessary localization calls.

US ENGLISH ALL THE WAY, BAYBEEEE 🇺🇸 🚀 🇺🇸 🚀